src/delta/delta.c

```diff
@@ -184,6 +184,10 @@ static int found_override(const char *top, const char *bottom) {
         return r;
 }
 
+DEFINE_PRIVATE_HASH_OPS_FULL(enumerate_dir_hash_ops_free_free,
+                             char, string_hash_func, string_compare_func, free,
+                             char, free);
```

Comment:
no need to define a the hash ops for this specific case. we already have a generic definition for this combination of callbacks: `string_hash_ops_free_free`, which you can just use here.